### ISSUE TITLE
Pressable onPress event is triggered when prop is disable

### ISSUE THREAD

**Issue by cgoldsby** [react-native-tvos/react-native-tvos#550]
### Description

According to the React Native [documentation](
https://reactnative.dev/docs/pressable#disabled), the `disabled` prop prevents the onPress event.
> Whether the press behavior is disabled.

The implementation for tvOS does not match the documentation requirement. When a Pressable component has `disabled` set to `true`, an `onPress` handler is still called.

I sense that the new if-statement for the tvOS platform should also check if the component is not disabled for select and longSelect events.

https://github.com/react-native-tvos/react-native-tvos/blob/c266ddb742c1d3694e1b86fd2f8bbf466a9720d8/Libraries/Components/Pressable/Pressable.js#L393

https://github.com/react-native-tvos/react-native-tvos/blob/c266ddb742c1d3694e1b86fd2f8bbf466a9720d8/Libraries/Components/Pressable/Pressable.js#L398

It is not entirely clear if the current implementation is functioning as designed; I would be more than happy to submit a PR.

### Version

0.71.12-0

### Output of `npx react-native info`

info Fetching system and libraries information...
System:
    OS: macOS 13.4.1
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 2.37 GB / 32.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 18.16.1 - /usr/local/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 9.8.1 - /usr/local/bin/npm
    Watchman: Not Found
  Managers:
    CocoaPods: 1.12.1 - /Users/cgoldsby/.rbenv/shims/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 22.4, iOS 16.4, macOS 13.3, tvOS 16.4, watchOS 9.4
    Android SDK: Not Found
  IDEs:
    Android Studio: 2022.2 AI-222.4459.24.2221.10121639
    Xcode: 14.3.1/14E300c - /usr/bin/xcodebuild
  Languages:
    Java: Not Found
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.2.0 => 18.2.0
    react-native: Not Found
    react-native-macos: Not Found
    react-native-tvos:  0.71.12-0
  npmGlobalPackages:
    *react-native*: Not Found
info React Native v0.72.3 is now available (your project is running on v0.71.12-0).
info Changelog: https://github.com/facebook/react-native/releases/tag/v0.72.3
info Diff: https://react-native-community.github.io/upgrade-helper/?from=0.71.12-0
info For more info, check out "https://reactnative.dev/docs/upgrading".

### Steps to reproduce

Create a Pressable component that is disabled and has an onPress handler. The expectation is that "Pressed" is not logged to the console.

### Snack, code example, screenshot, or link to a repository

```jsx
<Pressable disabled={true} onPress={() => console.log('Pressed')} style={{backgroundColor: 'red', width: 100, height: 100}}/>
```

**Comment by douglowder** [comment]
Confirmed, `Pressable` seems to be ignoring the `disabled` prop. This looks like a regression. Investigating.

**Comment by douglowder** [comment]
In fact, if the component has `disabled=true`, then it should not be focusable or selectable. So both `isTVSelectable` and `focusable` should be false.

**Comment by douglowder** [comment]
Fixed in release 0.71.13-1.

**Comment by sharan-SKY** [comment]
i believe there is another issue where longSelect gets triggered even when I do a normal press, is there an existing issue for this? I am using version 0.68.2
